### PR TITLE
Did more algo 

### PR DESCRIPTION
**Pull request by jacgit18**

### CODE REVIEW

**Comment by jacgit18** [comment]
fixed some of the issues on past problems and went through some of the recent problems

File: joshuaCarpentier/assignments/fastSlow/lc141/lc141.js [hunk 1]
```diff
@@ -35,24 +81,6 @@ head.next.next.next.next.next.next = head.next.next.next
...
-//     fast = fast.next.next;
-//     slow = slow.next;
-//     if (slow === fast) {
-//       return true; // found the cycle
-//     }
-//   }
-//   return false;
-// }
-// -----
-
```
Comment (cbluiux) [inline joshuaCarpentier/assignments/fastSlow/lc141/lc141.js:55]: Why was this solution removed?

File: joshuaCarpentier/assignments/fastSlow/lc141/lc141.js [hunk 2]
```diff
@@ -14,18 +48,30 @@ class Node {
...
+        fast = fast.next.next;
+        slow = slow.next;
+        if (slow === fast){
+          return true;
+        }
+      }
+
+// dont necessarly need to have else when just returning boolean value
   return false
 }
```
Comment (cbluiux) [inline joshuaCarpentier/assignments/fastSlow/lc141/lc141.js:64]: Code looks good! Small feedback would be not to leave comments in your codebase unless absolutely necessary & where is the love for typing it twice?!? #LEGGO #GoodWork #PracticeHowYouPlay #TypeTwice

File: joshuaCarpentier/assignments/fastSlow/lc141/lc141.js (hunk 1 above)
Comment (jacgit18) [reply joshuaCarpentier/assignments/fastSlow/lc141/lc141.js:55]: I reversed engineered it in the function

**Pull request merged**